soh/soh/Enhancements/randomizer/3drando/hint_list/hint_list_item.cpp

```diff
@@ -1939,6 +1939,107 @@ void HintTable_Init_Item() {
                        Text{"a Piece of the Triforce", /*french*/"un fragment de la Triforce", /*spanish*/"un fragmento de la Trifuerza"}
     );
 
+    hintTable[GOHMA_BOSS_SOUL] = HintText::Item({
+                      //obscure text
```

Comment:
nit: something funky happened here with the whitespace

Reply:
Will be addressed once larger logic issues are resolved.